### ISSUE TITLE
Test on python 3.8

### ISSUE THREAD

**Issue by psilentp** [FlyRanch/figurefirst#56]
Issue originally pointed out in #55 by @clbarnes.

**Comment by clbarnes** [comment]
I'm intrigued - the Actions config doesn't give the name of a tox env specified in the tox config, and the tox config doesn't include 3.8, but Actions does. How does that all work?

Either way, I think that explicitly installing the dependencies outside of tox isn't necessary - tox will install the dependencies in a fresh environment itself.

**Comment by psilentp** [comment]
Yes, that was sloppy on my part --  the build and test are redundant.

**Comment by psilentp** [comment]
It might also be useful to setup the CI to publish tagged releases.  https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

**Comment by psilentp** [comment]
> I'm intrigued - the Actions config doesn't give the name of a tox env specified in the tox config, and the tox config doesn't include 3.8, but Actions does. How does that all work?

I think to answer your question I think this bit of trickery suggested by the github actions tutorial is supposed line up the environments. 

https://github.com/FlyRanch/figurefirst/blob/c8cad3d18f2eaa62ed418649918bc3c0fbdaf252/.github/workflows/continuous-integration-workflow.yml#L29

Not sure if this is the best approach.
